localization/languages/pt-BR.json

```diff
@@ -138,9 +138,9 @@
         "settingsHistoryButtonToggle": "Habilitar botão voltar",
         "settingsAdditionalFeaturesHeading": "Recursos adicionais",
         "settingsUserscriptsToggle": "Habilitar scripts do usuário",
-        "settingsSeparateTitlebarToggle": null, //missing translation
+        "settingsSeparateTitlebarToggle": "Habilitar modo compacto",
```

Comment:
I'm not sure this matches the meaning of the setting. When the setting is disabled, the buttons are all in one line:

<img src="https://user-images.githubusercontent.com/15659999/78828973-f1280b00-79bb-11ea-8f33-36431b1d28ea.png"/>

When it's enabled, the buttons and tabs are on two separate lines:

<img src="https://user-images.githubusercontent.com/15659999/76693351-361d8380-6642-11ea-9889-390d2b5eaee3.png">

This translation would mean the opposite if I'm understanding it correctly.

Reply:
Confused kkkk, **I still don't know how to solve this item, what do you think of** (Disable compact mode) for now?

**Second google translator**

Reply:
The meaning of "use separate titlebar" is that when it's enabled, the title bar is separated from the tab bar - is it possible to translate it like that?